Tests/ReactiveSwiftTests/SignalProducerSpec.swift

```diff
@@ -2696,6 +2721,6 @@ extension SignalProducer {
 			}
 		}
 
-		return SignalProducer.attempt(operation)
+		return SignalProducer(operation)
```

Comment:
[A test case](https://github.com/ReactiveCocoa/ReactiveSwift/pull/343/files#diff-9cd786f9e52e786771d96cf8c42d3befR207) was added specifically for this, and this set of overloads doesn't appear to be ambiguous.

Moreover, it is expected to be called with trailing closures anyway. If it is ambiguous as trailing closures, adding labels do not help.